x-pack/docs/en/ilm/apis/move-to-step.asciidoc

```diff
@@ -0,0 +1,112 @@
+[role="xpack"]
+[testenv="basic"]
+[[ilm-put-policy]]
+=== Move To Step API
+++++
+<titleabbrev>Move To Step</titleabbrev>
+++++
+
+Moves a managed index into a specific execution step its policy
```

Comment:
Needs a note tagging this as an expert API that should only be used if there are no other alternatives